linuxrc.c

```diff
@@ -1278,6 +1278,13 @@ void lxrc_init()
 
   util_update_kernellog();
 
+  if(config.loghost)
+  {
+    char cmd[40];
+    snprintf(cmd, sizeof(cmd), "/scripts/remote_log_setup %s", config.loghost);
```

Comment:
I agree that the length of the string should be increased. However I'd prefer allocating in the stack with something like `char cmd[100];`. Any reason to prefer the dynamic allocation?